cloud/pkg/cloudstream/containerexec_connection.go

```diff
@@ -41,10 +42,15 @@ func (c *ContainerExecConnection) GetMessageID() uint64 {
 }
 
 func (c *ContainerExecConnection) SetEdgePeerDone() {
-	close(c.edgePeerStop)
```

Comment:
Seems haven't found that where close the `edgePeerStop` channel?

Reply:
The memory of channel `edgePeerStop` can be released by the automatic memory management of golang when the exec connection is down.

Reply:
okey.